### DISCUSSION TITLE
Getting error for WolframAlphaQueryRun

### DISCUSSION THREAD

**Discussion by sebastian3bner** [langflow-ai/langflow#355] [Q&A]
Hi,

I'm getting the following error:

```
Error building node WolframAlphaAPIWrapper: 1 validation error for WolframAlphaAPIWrapper
__root__
Did not find wolfram_alpha_appid, please add an environment variable `WOLFRAM_ALPHA_APPID` which contains it, or pass `wolfram_alpha_appid` as a named parameter. (type=value_error)
```

I've added my wolfram app-id as a local environment variable.

**Comment by carlosrcoelho** [comment]
Hey!

Thanks for sharing your question here. I'm sorry for not getting back to you sooner.
Langflow has gone through some major changes since then. I recommend trying out the new version for the best experience. If you have any issues, please open a new one; we'll be here to help.
